src/client.js

```diff
@@ -33,6 +34,7 @@ function Client(options) {
 		this.use(resizer(options.resizerOptions));
 	}
 	this.use(userActivityEvents);
+	this.use(syncIntl);
```

Comment:
Can we have this behind an option (default to true)? `frau-jwt` for instance [creates a client super brielfy](https://github.com/Brightspace/frau-jwt/blob/master/framed.js#L12) (and forgets to close it I guess... oops?), and doesn't need to participate in this stuff, and definitely not wait on other requests.

Ideally user activity events would also be behind a default to true option

Reply:
Yeah I can do that. Then I can bring that other test back and disable the plugin.